package.json

```diff
@@ -24,7 +24,8 @@
     "async": "*",
     "moment": "^2.19.2",
     "request": "*",
-    "request-promise-native": "*"
+    "request-promise-native": "*",
+    "al-collector-js": "git://github.com/alertlogic/al-collector-js#master"
```

Comment:
We don't use `al-collector-js` dependency anywhere. Can you remove it pls?

Reply:
+1, let's add it back when needed.